docs/api/utils/partitionByKey.md

```diff
@@ -28,6 +28,42 @@ export function partitionByKey<T, K, R>(
 
 2. A stream with the list of active keys
 
+### Examples
+
+```ts
+const source = interval(1000);
+const [getGroupByKey, keys$] = partitionByKey(
+  source,
+  x => x % 2 == 0 ? "even" : "odd",
+  (groupedObservable$, key) => groupedObservable$.pipe(map(x => `${x} is ${key}`))
+);
```

Comment:
>  I think that this example could be useful, because it's straightforward and it shows how the parameters are used

That was my thinking. If you want it out of there, or don't want it to be the first example, you need only say so!

As for the main use case, I'm on it. My local file says:

    A more typical use:
    ```
      (coming soon)
    ```